### PR TITLE
metrics: add Quantile Score and Quantile Skill Score

### PR DESCRIPTION
**Pull request by dplarson**
Add documentation to the Metrics page regarding Quantile Score and Quantile Skill Score, which are being added via PR https://github.com/SolarArbiter/solarforecastarbiter-core/pull/530

### CODE REVIEW

**Comment by dplarson** [comment]
Here's a screenshot of the Metrics page with quantile score and quantile skill score added:

Screenshot_2020-08-04 Metrics Solar Forecast Arbiter

File: metrics.md [hunk 1]
```diff
@@ -295,11 +295,52 @@ Uncertainty is the variance of the event indicator $$ o(t) $$. Low values of UNC
...
+
+If instead $$ \text{obs} < \text{fx} $$, then QS is also non-negative:
+
+$$\begin{align}
+    (\text{fx} - \text{obs}) &> 0 \\
+    (p - \mathbf{1}\{\text{obs} > \text{fx}\}) &= (p - 0) \geq 0 \\
+    (\text{fx} - \text{obs}) \cdot p &\geq 0 \\
+\end{align} $$
+
+Based on the above definitions, smaller QS values indicate more accurate forecasts.
```
Comment (wholmgren) [inline metrics.md:332]:
```suggestion
Smaller QS values indicate more accurate forecasts.

QS is always greater than or equal to 0. If $$ \text{obs} > \text{fx} $$, then QS is non-negative:

$$\begin{align}
    (\text{fx} - \text{obs}) &< 0 \\
    (p - \mathbf{1}\{\text{obs} > \text{fx}\}) &= (p - 1) \leq 0 \\
    (\text{fx} - \text{obs}) \cdot (p - 1) &\geq 0
\end{align} $$

If instead $$ \text{obs} < \text{fx} $$, then QS is also non-negative:

$$\begin{align}
    (\text{fx} - \text{obs}) &> 0 \\
    (p - \mathbf{1}\{\text{obs} > \text{fx}\}) &= (p - 0) \geq 0 \\
    (\text{fx} - \text{obs}) \cdot p &\geq 0 \\
\end{align} $$
```

Comment (wholmgren) [reply metrics.md:332]: To me, the most important thing is that smaller QS is better. The second most important thing is that it's >=0. The proof is the least interesting. So my suggestion reorders the text to match that. Feel free to disagree.

Comment (dplarson) [reply metrics.md:332]: That makes sense to me.

File: metrics.md [hunk 2]
```diff
@@ -295,11 +295,52 @@ Uncertainty is the variance of the event indicator $$ o(t) $$. Low values of UNC
...
 
-$$ BSS = 1 - \frac{\text{BS}_f}{\text{BS}_{\text{ref}}} $$
+$$ \text{BSS} = 1 - \frac{\text{BS}_f}{\text{BS}_{\text{ref}}} $$
 
 where $$ \text{BS}_f $$ is the BS of the forecast of interest, and $$ \text{BS}_{\text{ref}} $$ is the BS of the reference forecast. BSS greater than zero indicates the forecast performed better than the reference and vice versa for BSS less than zero, while BSS equal to zero indicates the forecast is no better (or worse) than the reference.
 
 
+### Quantile Score (QS) {#qs}
+{: .anchor }
+QS measures the accuracy of quantile forecasts, in which the forecast predicts the variable value corresponding to a constant probability ([Koenker78](#ref-koenker78), [Wilks11](#ref-wilks11)). QS is similar to BS, but measures accuracy in terms of the variable value (e.g. MW) and is defined as:
```
Comment (wholmgren) [inline metrics.md:305]: I believe quantile score is only in the 4th edition of Wilks. Easiest thing is probably to add an new reference for that and leave the Wilks 2011 reference alone. Or we can double check that all of the existing references to Wilks remain good in the new edition and replace it.

Comment (dplarson) [reply metrics.md:305]: If it's fine with you, let's just start with adding the 4th edition as a separate reference.

**Pull request merged**